Integration/AndroidAAR/BabylonNative/src/main/cpp/BabylonNativeJNI.cpp

```diff
@@ -0,0 +1,170 @@
+#include <jni.h>
+#include <stdlib.h>
+#include <string.h>
+#include <signal.h>
+#include <time.h>
+#include <memory>
+#include <android/native_window.h> // requires ndk r5 or newer
+#include <android/native_window_jni.h> // requires ndk r5 or newer
+#include <android/log.h>
+#include <Babylon/Console.h>
+#include <Babylon/RuntimeAndroid.h>
+#include <android/asset_manager.h>
+#include <android/asset_manager_jni.h>
+#include <InputManager.h>
+
+extern "C" {
+JNIEXPORT void JNICALL Java_BabylonNative_Wrapper_initEngine(JNIEnv* env, jobject obj, jobject assetMgr);
+JNIEXPORT void JNICALL Java_BabylonNative_Wrapper_finishEngine(JNIEnv* env, jobject obj);
+JNIEXPORT void JNICALL Java_BabylonNative_Wrapper_surfaceCreated(JNIEnv* env, jobject obj, jobject surface);
+JNIEXPORT void JNICALL Java_BabylonNative_Wrapper_activityOnPause(JNIEnv* env);
+JNIEXPORT void JNICALL Java_BabylonNative_Wrapper_activityOnResume(JNIEnv* env);
+JNIEXPORT void JNICALL Java_BabylonNative_Wrapper_surfaceChanged(JNIEnv* env, jobject obj, jint width, jint height, jobject surface);
+JNIEXPORT void JNICALL Java_BabylonNative_Wrapper_loadScript(JNIEnv* env, jobject obj, jstring path);
+JNIEXPORT void JNICALL Java_BabylonNative_Wrapper_eval(JNIEnv* env, jobject obj, jstring source, jstring sourceURL);
+JNIEXPORT void JNICALL Java_BabylonNative_Wrapper_setTouchInfo(JNIEnv* env, jobject obj, jfloat dx, jfloat dy, jboolean down);
+};
+
+std::unique_ptr<Babylon::RuntimeAndroid> runtime{};
+std::unique_ptr<InputManager::InputBuffer> inputBuffer{};
+
+static AAssetManager* g_assetMgrNative = nullptr;
```

Comment:
Move these statics into the anonymous namespace?

Reply:
Any opinion on that point @bghgary ?
Anonymous namespace is not used in other testapp.
I'm fine with ou without.

Reply:
I don't have a strong preference, but I would prefer not to mix static and anonymous namespace.

Reply:
static removed.